src/QMCHamiltonians/OperatorBase.cpp

```diff
@@ -42,7 +42,6 @@ OperatorBase::OperatorBase() : myIndex(-1), Dependants(0), Value(0.0), tWalker(0
  */
 void OperatorBase::mw_evaluate(const RefVector<OperatorBase>& O_list, const RefVector<ParticleSet>& P_list)
 {
-#pragma omp parallel for
```

Comment:
Do not change this. This is needed so far. I verified that this works very well with OMP_NUM_THREADS=16 and 1 crowd in the input.

Reply:
So now this is OMP_NUM_THREADS=1,16 behavior but induced by forcing one crowd? I'll put this back in for now (as I expect this has to do with AMD support), but  we need to continue to abstract threading and the actual runtime thread setup needs to be handled in one place and not via a proprietary colleciton of ENV. The adhoc nature of OMP thread safety and its opaqueness when reading the source and compiling time is going to continue to be an issue until we do.

Reply:
If running 1 crowd is requested, the first OpenMP level is an inactive threading level, there is no need to set nested threading. When using a list of thread numbers like OMP_NUM_THREADS=1,16, they refer to active threading levels. I used this on Summit to compare with the CUDA code which is conceptually 1 crowd.

Reply:
I get what you mean about the semantics of openmp but this throws threading control back to implicit openmp threading.  In application threading model we should be using to reason about thread scopes and synchronization this is a 1,16

Reply:
We can discuss in person with more details. This raw OpenMP implementation doesn't ignore the crowd level concurrency. Both crowd level and walker level are needed.